components/more-less/more-less.js

```diff
@@ -107,6 +102,23 @@ class MoreLess extends LocalizeMixin(LitElement)  {
 			}
 		};
 
+		for (let i = 0; i < langs.length; i++) {
+			if (langResources[langs[i]]) {
+				return {
+					[langs[i]]: langResources[langs[i]]
+				};
```

Comment:
Does it need to return a hash -- could it just return `langResources[langs[i]]`?

Reply:
I think we need the language though for `IntlMessageFormat` (https://github.com/BrightspaceUI/core/blob/master/mixins/localize-mixin.js#L240)

Reply:
Ah I see. To make it more clear, maybe it could be something like this:

```
return {
    lang: langs[i],
    resources: langResources[langs[i]]
};
```

That way we could also ask them to return other stuff in the future if we need it.